pkg/ccl/changefeedccl/changefeedbase/errors.go

```diff
@@ -116,7 +117,7 @@ func IsRetryableError(err error) bool {
 		return true
 	}
 
-	return joberror.IsDistSQLRetryableError(err)
+	return (joberror.IsDistSQLRetryableError(err) || flowinfra.IsNoInboundStreamConnectionError(err))
```

Comment:
I kinda think that we should add this to joberror somewhere -- see request from @adityamaru (on the issue) to also make builio aware of this error